src/index.html

```diff
@@ -2,17 +2,46 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1.0"
-    />
-    <title>Product cards</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Product card</title>
+    <link rel="stylesheet" href="./styles/index.scss" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
     <link
-      rel="stylesheet"
-      href="./styles/index.scss"
-    />
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet" />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <section class="product-card" data-qa="card">
+      <img src="./images/imac.jpeg" alt="APPLE A1419 iMac 27 Retina 5K Monoblock" class="product-card__img" />
+
+      <div class="product-card__details">
+        <h3 class="product-card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+        <p class="product-card__sku">Product code: 195434</p>
+      </div>
+
+      <div class="product-card__rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <span class="product-card__reviews">Reviews: 5</span>
+      </div>
+
+      <div class="product-card__pricing">
+        <span class="product-card__pricing-label">Price:</span>
+        <span class="product-card__pricing-amount">$2,199</span>
+      </div>
+
+      <a href="#" class="product-card__button" data-qa="hover">
```

Comment:
if you have 3 or more attributes, moved them according to a good example
<img width="1117" alt="Знімок екрана 2024-08-14 о 14 49 45" src="https://github.com/user-attachments/assets/ecb75d4f-429d-4956-82a3-6ec49ae4de37">
```suggestion
      <a 
        href="#" 
        class="product-card__button" 
        data-qa="hover"
      >
```